mesh/mesh.go

```diff
@@ -118,25 +119,24 @@ func (msh *Mesh) recoverFromDB(latest types.LayerID) {
 
 	lyr, err := layers.GetProcessed(msh.cdb)
 	if err != nil {
-		msh.logger.With().Fatal("failed to recover processed layer", log.Err(err))
+		msh.logger.Fatal("failed to recover processed layer", zap.Error(err))
 	}
 	msh.processedLayer.Store(lyr)
 
 	applied, err := layers.GetLastApplied(msh.cdb)
 	if err != nil {
-		msh.logger.With().Fatal("failed to recover latest applied layer", log.Err(err))
+		msh.logger.Fatal("failed to recover latest applied layer", zap.Error(err))
 	}
 	msh.setLatestLayerInState(applied)
 
 	if applied.After(types.GetEffectiveGenesis()) {
 		if err = msh.executor.Revert(context.Background(), applied); err != nil {
-			msh.logger.With().
-				Fatal("failed to load state for layer", msh.LatestLayerInState(), log.Err(err))
+			msh.logger.Fatal("failed to load state for layer", msh.LatestLayerInState().Field().Zap(), zap.Error(err))
```

Comment:
NIT: either the `Field` method on `LayerID` needs to be updated to return a `zap.Field` or we should do something like this instead:

```suggestion
			msh.logger.Fatal("failed to load state for layer",
				zap.Uint32("layer_id", msh.LatestLayerInState().Uint32()),
				zap.Error(err),
			)
```